client/view/index/Index.html

```diff
@@ -119,11 +131,73 @@ <h2>Try an example deck</h2>
 	padding: 0 4px;
 }
 
+/*Start delete a seen workbook X*/
+.delete-seenWorkbook-x {
+	color: black;
+	font-weight: bold;
+	font-size: 20px;
+	cursor: pointer;
+	transition: 0.4s;
+	position: relative;
+	bottom: -1px;
+	padding-left: 10px;
+	user-select: none;
+	opacity: 1;
+}
+.delete-seenWorkbook-x:hover {
+	color: red;
+}
+@media (min-width: 960px) {
+	.delete-seenWorkbook-x {
+		opacity: 0;
+	}
+	li:hover .delete-seenWorkbook-x {
+		opacity: 1;
+	}
+}
+/*End delete a seen workbook X*/
+
+/*Start 'Are you sure?' window*/
+.are-you-sure-window {
+	position: fixed;
+	top: 0;
+	right: 0;
+	left: 0;
+	bottom: 0;
+	z-index: 100;
+	display: none;
+	background-color: rgba(0, 0, 0, 0.5);
+	opacity: 0;
+	transition: 0.4s;
+}
+.are-you-sure-window div {
+	animation: animatedown 0.4s ease-out;
+	position: relative;
+	top: 50px;
+	max-width: 600px;
+	margin: auto;
+	background-color: white;
+	padding: 20px;
+	border-radius: 4px;
+	box-shadow: 0 2px 5px 0 rgba(0,0,0,0.16),0 2px 10px 0 rgba(0,0,0,0.12);
+}@keyframes animatedown{
+	from {
+		opacity: 0.5;
+		top: -300px;
+	}
+	to {
+		opacity: 1;
+		top: 50px;
+	}
+}
+/*End 'Are you sure?' window*/
+
 </style>
 
 <script>
 import exampleWorkbooks from 'lib/example-workbooks.js'
 import SpreadsheetInput from 'lib/component/SpreadsheetInput.html'
+import smallIndexedDb from 'small-indexeddb';
```

Comment:
I want to try to avoid having the "view" layer (the code that describes the user interface) interacting directly with databases, or external servers.  It makes the views a lot simpler to work with.

These views do have access to the mediator thanks to these lines:

https://github.com/TehShrike/memorize-text-app/blob/b1b41e42bdaa0089ac2c846a5fd0838596a45963/client/index.js#L13-L20

which means that you can call `this.mediatorCall('forgetWorkbook', workbook.key)` in the methods down below.